### DISCUSSION TITLE
Domain expert - I'd like to help

### DISCUSSION THREAD

**Discussion by MattCantor** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#295] [General]
Hi all. I'm a partner in the executive compensation and employee benefits group at Fenwick & West, one of the founding members of the OCT coalition.  I just recently discovered this project and I'd very much like to help.

I deal with stock options and cap tables every day.  I'd be happy to lend any insight I can into the intricacies of stock options, RSUs, and other equity-based compensation awards.  Once I get a better understanding of the lay of the land here, I'd be happy to assist with documentation as well.

If anyone has ideas for how I can be useful or would just like to talk shop, please let me know!

**Comment by JSv4** [comment]
Hi @MattCantor, great to hear from you (and awesome to see a fellow attorney with a GitHub handle :+1: ). OCF is meant to be a data exchange standard between different tools and platforms, so you can use cap table data in a document automation tool or move from one platform to another, just as a few examples. Of course, having domain experts weigh in on both the standard and actually use it would be amazing too. 

We're currently writing some developer guides on how to achieve various common tasks in OCF - e.g. how to model a transaction or how to create a stock plan and issue an option. If you're familiarizing yourself with the standard anyway, if you happen to take a stab at working through some of the common company lifecycle events and want to write up a guide, that'd be super appreciated. If you commit a PR, you're more than welcome to present it at the Technical Working Group meeting.
